examples/api/src-tauri/src/lib.rs

```diff
@@ -71,6 +71,11 @@ pub fn run() {
                     .decorations(false);
             }
 
+            #[cfg(target_os = "macos")]
```

Comment:
Why not windows as well?

Reply:
Windows is already handled on the block above.